app/src/main/java/com/zeapo/pwdstore/crypto/PgpActivity.kt

```diff
@@ -831,13 +831,13 @@ class PgpActivity : AppCompatActivity(), OpenPgpServiceConnection.OnBound {
 
     private fun copyUsernameToClipBoard(username: String) {
         val clip = ClipData.newPlainText("pgp_handler_result_pm", username)
-        clipboard.primaryClip = clip
+        clipboard.setPrimaryClip(clip)
         showSnackbar(resources.getString(R.string.clipboard_username_toast_text))
     }
 
     private fun copyOtpToClipBoard(code: String) {
         val clip = ClipData.newPlainText("pgp_handler_result_pm", code)
-        clipboard.primaryClip = clip
+        clipboard.setPrimaryClip(clip)
```

Comment:
Why not property access syntax ?

Reply:
See the second paragraph of the PR description, this is because setPrimaryClip is now annotated as accepting a NonNull parameter, which breaks Kotlin's automated property generation.

Reply:
Oh shoot, missed it.